chart/values.yaml

```diff
@@ -59,6 +59,7 @@ images:
   airflow:
     repository: ~
     tag: ~
+    digest: ~
```

Comment:
Can you please add the comment that adding digest will override the tag ? I think it is not obvious what happens when someone specifies both.

Reply:
Definitely, will do.